paperslicer/utils/exports.py

```
@@ -0,0 +1,134 @@
+    _ensure_dir(out_dir)
+    csv_path = os.path.join(out_dir, f"images_summary_{datetime.now().strftime('%Y%m%d_%H%M')}.csv")
+    rows: List[Dict[str, object]] = []
+    for name in sorted(os.listdir(meta_dir)):
+        if not name.endswith('.json'):
+            continue
+        path = os.path.join(meta_dir, name)
+        try:
+            with open(path, 'r', encoding='utf-8') as f:
+                d = json.load(f)
+        except Exception:
+            continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)